common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,9 +244,38 @@ class ModuleI18nService(object):
     i18n service.
 
     """
-
-    def __getattr__(self, name):
-        return getattr(django.utils.translation, name)
```

Comment:
I checked all the XBlocks we know about: they use `ugettext`, `ungettext`, `strftime`, and with an attribute check, `get_language_bidi`.